src/components/ChatList/ChannelItem.style.ts

```diff
@@ -1,22 +1,22 @@
 import {StyleSheet} from 'react-native';
 
 import dimen from '../../utils/dimen';
-import {colors} from '../../utils/colors';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to import `COLORS` from the `theme` module instead of importing `colors` from the `colors` module. This change is consistent with the PR's goal of standardizing color usage across the application.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->